src/main/scala/com/vngrs/etl/transformers/JsonToRowTransform.scala

```diff
@@ -0,0 +1,25 @@
+package com.vngrs.etl.transformers
+
+import com.vngrs.etl.Transform
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.{Row, SQLContext}
+
+/**
+  * Transform `JSON` formatted [[String]]s to [[org.apache.spark.sql.Row]]s.
+  */
+final case class JsonToRowTransform() extends Transform[String, Row] {
```

Comment:
I thought that as well, but most of the plugins will require parameter therefore they will be case classes. So, I thought that all plugins should have similar syntax. If you are saying making object is a better choice, I can change it.